student.rb

```diff
@@ -0,0 +1,15 @@
+require './person'
+require './classroom'
+
+class Student < Person
+  belongs_to :classroom
+
+  def initialize(age, classroom, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
+    @classroom = classroom
+  end
```

Comment:
- [ ] It will be great to create a both-way relationship. Remember the classroom and the student have a both-way relationship. Kindly refer to the article I have given above.